Assets/Scripts/LevelSelect/Camera/VehicleCamera.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace DN.Levelselect.CameraMovement
+{
+    /// <summary>
+    /// THis is the camera movement to follow the car
+    /// </summary>
+    public class VehicleCamera : MonoBehaviour
+    {
+
+        [SerializeField] private Transform cameraRig;
+
+        [SerializeField] [Range(1, 20)] private float followSpeed = 16;
+
+        private Vector3 cameraPositionOffset;
+
+        private Camera camera;
+
+        void Awake()
+        {
+            cameraPositionOffset = cameraRig.localPosition;
+            camera = cameraRig.GetChild(0).GetComponent<Camera>();
+        }
+
+        void FixedUpdate()
+        {
+            cameraRig.position = Vector3.Lerp(cameraRig.position, transform.position + cameraPositionOffset, Time.deltaTime * followSpeed);
+        }
```

Comment:
Gooi dit in een `LateUpdate` ipv een `FixedUpdate`, dan wordt het elke frame uitgevoerd nadat de auto zich bewogen heeft.

Reply:
de auto zichzelf rijdt ook in FixedUpdate dus misschien niet nodig maar het kan er smoother uit zien.